portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+  const QUOTES =
+      ['Toit Nups!',
+       'NINE-NINE!',
+        'Cool cool cool cool cool, no doubt no doubt.',
+         'Why is no one having a good time? I specifically requested it.'];
+
 /**
  * Adds a random quote from Brooklyn-99 to the page.
  */
 function addRandomQuote() {
-  const quotes =
-      ['Toit Nups!', 'NINE-NINE!', 'Cool cool cool cool cool, no doubt no doubt.', 'Why is no one having a good time? I specifically requested it.'];
-
   // Pick a random greeting.
-  const quote = quotes[Math.floor(Math.random() * quotes.length)];
+  const quote = QUOTES[Math.floor(Math.random() * QUOTES.length)];
 
   // Add it to the page.
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+
+
+/**
+ * This function opens the tab we want to access and hides the unnecessary tabs
+ */
```

Comment:
When we write javascript internally, we use the closure type system
https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System

You're not using the closure compiler, so nothing will enforce these types, but its a good idea to start getting used to using them anyway.
For this function, you'd want to include types like this:
```
/**
 * @param {!Event} evt The click event triggering this action
 * @param {string} tabName The name of the tab to open
 */
```